packages/nala/build.sh

```diff
@@ -32,42 +26,20 @@ termux_step_pre_configure() {
 		${_CROSSENV_PREFIX}
 	popd
 	. ${_CROSSENV_PREFIX}/bin/activate
-
-	build-pip install tomli
 }
 
 termux_step_make_install() {
-	python setup.py install --force
-
 	export PYTHONPATH=$TERMUX_PREFIX/lib/python${_PYTHON_VERSION}/site-packages
-	python setup.py install --force --prefix $TERMUX_PREFIX
-
-	pushd $PYTHONPATH
-	_NALA_EGG=
-	for f in nala-${TERMUX_PKG_VERSION##*:}-py${_PYTHON_VERSION}.egg; do
-		if [ -e "$f" ]; then
-			_NALA_EGG="$f"
-			break
-		fi
-	done
-	test -n "${_NALA_EGG}"
-	popd
+	python -m pip install . --force --prefix $TERMUX_PREFIX
```

Comment:
Better to use `python${TERMUX_PKG_VERSION}` and `pip${TERMUX_PKG_VERSION}` here

Reply:
In this case it would just be `python${_PYTHON_VERSION} -m pip (...)`, without pip version, since we are calling the module, not the executable